mautic/mautic
# Add contact segment membership webhook

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 




#### Description:

This PR just add Contact Segment Membership Change Event to Webhooks

![image](https://user-images.githubusercontent.com/462477/136180165-fc1d916a-8518-4741-a38a-f802381e5b0b.png)




#### Steps to test this PR:


1. Load up [this PR](https://mautibox.com)
2. Go to https://webhook.site/ and copy link of your webhook
3. Go to Mautic and Webhooks
4. Create  Contact Segment Membership Change Event to your webhook.site url
5. Go to contact list and change segment membership for few contacts
6. Check If you receive correct data to webhook.site
